ssh-notes/using-ssh-keys-with-passwords.md

```diff
@@ -0,0 +1,84 @@
+# Using SSH Keys with Passwords
+
+When you create an SSH key, you are allowed to assign a password to it.  This means whenever you attempt to use the key it will prompt you with a password.  Some software, such as VS Code, canwill deal with this for you and cache the appropriate permissions it needs.
```

Comment:
```suggestion
When you create an SSH key, you are allowed to assign a password to it.  This means whenever you attempt to use the key it will prompt you with a password.  Some software, such as VS Code, can deal with this for you and cache the appropriate permissions it needs.
```